src/java.desktop/share/classes/javax/imageio/spi/IIORegistry.java

```diff
@@ -87,10 +83,10 @@
 public final class IIORegistry extends ServiceRegistry {
 
     /**
-     * A {@code Vector} containing the valid IIO registry
+     * An {@code ArrayList} containing the valid IIO registry
      * categories (superinterfaces) to be used in the constructor.
      */
-    private static final Vector<Class<?>> initialCategories = new Vector<>(5);
+    private static final ArrayList<Class<?>> initialCategories = new ArrayList<>(5);
```

Comment:
Should this be an immutable list and remove the static block?

```suggestion
    private static final List<Class<?>> initialCategories = List.of(
                ImageReaderSpi.class,
                ImageWriterSpi.class,
                ImageTranscoderSpi.class,
                ImageInputStreamSpi.class,
                ImageOutputStreamSpi.class);
```

Reply:
I think this one could be a better approach.

Reply:
Applied